cf/data/data.py

```diff
@@ -10082,30 +10082,57 @@ def swapaxes(self, axis0, axis1, inplace=False, i=False):
         d._set_dask(dx, reset_mask_hardness=False)
         return d
 
-    def fits_in_memory(self, itemsize):
-        """Return True if the master array is small enough to be
-        retained in memory.
+    def fits_in_memory(self):
+        """Return True if the array is small enough to be retained in
+        memory.
+
+        Returns True if the size of the array with all delayed
+        operations computed, including a space for a full boolean
+        mask, is small enough to be retained in available memory.
+
+        The available memory is the actual free memory less the amount
+        reserved as a temporary work space, i.e. ``cf.free_memory() -
+        cf.fm_threshold()``
+
+        The delayed operations are actually not computed by
```

Comment:
Perhaps this merits a `.. warning::` emphasis? (If you don't think so, it's fine as-is, though.)